changelogs/unreleased/gh-7593-console-debugger.md

```diff
@@ -0,0 +1,3 @@
+## feature/debugger
+
+* Introduced readline support to tarantool debugger (gh-7738).
```

Comment:
```suggestion
* Introduced `readline` support for the Tarantool debugger (gh-7738).
```

Reply:
This is actually a backport, so according to [this](https://github.com/tarantool/tarantool/pull/8880#discussion_r1266507421) discussion, no changes to changelog should be made. But it turns out, the original changelog slipped in unchecked, because I've modified it after it was approved and forgot to rerequest review.

Reply:
@andreyaksenov, please confirm that it is ok to leave as is.